static/app/utils/theme/theme.chonk.tsx

```diff
@@ -27,20 +27,20 @@ const formTheme: FormTheme = {
    */
   form: {
     md: {
-      height: 32,
-      minHeight: 32,
+      height: '32px',
+      minHeight: '32px',
```

Comment:
note: I had to change this to be able to use `fontSize` inside calc expressions. This is something I’ve discussed with @natemoo-re before, as we now have a mix between numbers and strings with `px`. We then concluded strings are preferred, and I agree because we can’t use them in calc expression otherwise.

Reply:
Yep, works for me! Technically we could use `calc(${height} * 1px)` but this is much easier to grok.

Reply:
Oh wow I didn't know that, TIL ❤️